src/utils/index.ts

```diff
@@ -0,0 +1,28 @@
+import type { OPERATION } from '../types';
+import { CONST } from './const';
+
+export function isOperation(order: OPERATION | string): order is OPERATION {
+  return CONST.OPERATION_LIST.some((operation) => operation === order);
+}
+
+export function calculate(
+  left: number,
+  operation: Omit<OPERATION, '='>,
+  right: number
+) {
+  switch (operation) {
+    case '+':
+      return left + right;
+    case '-':
+      return left - right;
+    case 'X':
+      return left * right;
+    case '/': {
+      if (right === 0) {
+        return CONST.INFINITY;
+      }
+      return Math.floor(left / right);
```

Comment:
대부분의 경우 나누는 값이 0이면, Infinity나 -Infinity가 반환되므로, `right === 0` 조건으로 Infinity로 판단한 것이 맞을 수 있습니다.

하지만, `0 / 0` 의 케이스도 존재합니다. `0 / 0`은 `Infinity`가 아닌 `NaN`입니다. `Infinity`와 `NaN` 모두 number 타입이지만 서로 다른 값이므로 요구사항을 엄격히 준수한다면 케이스를 나눌 필요가 있습니다.

Apple 계산기에서도 `0 / 0` 계산 결과 NaN이므로 `숫자 아님`이라고 표시되고, `1 / 0`이나 `-1 / 0`은 Infinity, -Infinity라서 `오류`로 달리 표시됩니다.

따라서 `left / right` 연산을 시킨 다음 그 결과에 대해 반환값 분기를 하는 것이 더 정확할 것 같아요. 😄 
```suggestion
    case '/': {
      const result = left / right;
      if (result === Infinity || result === -Infinity) {
        return '오류';
      }
      if (Number.isNaN(result)) {
        return '숫자 아님';
      }
      return Math.floor(result);
    }
```

Reply:
이부분에 대한 케이스는 고려를 못했습니다..
먼저 계산하고 이에 대해 분기해보도록 하겠습니다!!